tools/src/main/java/org/apache/rocketmq/tools/admin/DefaultMQAdminExt.java

```diff
@@ -128,12 +128,18 @@ public MessageExt viewMessage(
     }
 
     @Override
-    public QueryResult queryMessage(String topic, String key, int maxNum, long begin,
-        long end) throws MQClientException,
-        InterruptedException {
+    public QueryResult queryMessage(String topic, String key, int maxNum, long begin, long end)
+        throws MQClientException, InterruptedException {
+
         return defaultMQAdminExtImpl.queryMessage(topic, key, maxNum, begin, end);
     }
 
+    public QueryResult queryMessageByUniqueKey(String topic, String key, int maxNum, long begin, long end)
```

Comment:
The function name, queryMessageByUniqueKey is inconsistent with queryMessageByUniqKey, I suggest keep the same

Reply:
ok, I will fix it